alignak/http/generic_interface.py

```diff
@@ -224,9 +222,12 @@ def get_external_commands(self):
         :return: serialized external command list
         :rtype: str
         """
-        with self.app.external_commands_lock:
-            cmds = self.app.get_external_commands()
-            raw = serialize(cmds)
+        if hasattr(self.app, 'external_commands_lock'):
```

Comment:
It shoud be always true. If not then add it. The lock ensure we don't loose external commands. Maybe the if is on the get_external_commands method. How did you hit this case?